source/_components/sensor.fronius.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Fronius"
+description: "Instructions on how to connect your Fronius Inverter to Home Assistant."
+date: 2018-01-05 11:43:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Energy
+logo: fronius.png
+ha_iot_class: "Local Polling"
+ha_release: 0.92
+---
+
+The `fronius` sensor will poll a [Fronius](http://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors (or attributes of sensors) in Home Assistant.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
```

Comment:
Only required items in the first example, this is because users can do copy and paste.

Reply:
It does not really make sense to remove all entries from "monitored conditions" because there are no standard conditions and no required ones - an empty list does not really help when setting up the sensor

Reply:
If you remove `scope` and `device` from the example then it is okay.